### PR TITLE
Add Freshdesk Messaging (formerly Freshchat) Facade

### PR DESCRIPTION
**Pull request by coliff**
Repo: https://github.com/coliff/freshdesk-messaging-facade/
Demo: https://coliff.github.io/freshdesk-messaging-facade/

### CODE REVIEW

**Review by patrickhulce — APPROVED** [review]
awesome I love it! 🎉

File: data/entities.js [hunk 1]
```diff
@@ -579,7 +579,23 @@ module.exports = [
...
+        urlPatterns: ['wchat.freshchat.com'],
+        facades: [
+          {
+            name: 'Freshdesk Messaging (formerly Freshchat) Facade',
+            repo: 'https://github.com/coliff/freshdesk-messaging-facade/',
+          },
+        ],
+      },
+    ],
+  },
```
Comment (patrickhulce) [inline data/entities.js:598]:
```suggestion
```

**Comment by patrickhulce** [comment]
aw shoot sorry @coliff apparently there's already a separate entry for freshchat, maybe this belongs on that entity?

image

**Comment by coliff** [comment]
hey again @patrickhulce - yep, makes sense to move it to that section for now I think. [Freshchat is now just called Freshdesk Messaging](https://www.freshworks.com/live-chat-software/#:~:text=Freshdesk%20Messaging%20(formerly%20Freshchat)), but it's using the freshchat.com domain name still so I think it's fine to stay in the Freshchat entry for now.

**Comment by patrickhulce** [comment]
heads up @adamraine @connorjclark new facade here

**Comment by coliff** [comment]
thanks @patrickhulce - I spent a lot of time working on this and I'm very happy for it to be included in this project 😊

**Pull request merged**